BuildResidentialScheduleFile/resources/schedules.rb

```diff
@@ -77,6 +77,8 @@ def create(args:)
       @column_names = SchedulesFile.ColumnNames
     end
 
+    return true if args[:geometry_num_occupants] == 0
+
```

Comment:
This isn't needed because the measure will exit earlier due to your measure.rb change, no?